doc/content/design/RDP.md

```diff
@@ -0,0 +1,98 @@
+---
+title: RDP control
+layout: default
+design_doc: true
+revision: 2
+status: released (XenServer 6.5 SP1)
+design_review: 12
```

Comment:
I see this field in the original [page](https://xapi-project.github.io/features/RDP.html) is a link to: http://github.com/xapi-project/xapi-project.github.io/issues/12, do we need to keep it?

Reply:
Yes, I'll add those links, just for historical reasons.